README.md

```diff
@@ -472,6 +488,159 @@ async function saveCoverageData (codeBlocks) {
 export default runUnitTests
 ```
 
+
+## running linter
+```js
+// runLinter.js
+import fs from 'fs'
+import saveToFile from './saveToFile'
+import padRight from 'lodash/padEnd'
+import flatten from 'lodash/flatten'
+import { CLIEngine } from 'eslint'
+import Table from 'cli-table'
+
+export const countLinterErrors = (results) => results[0].messages.length
+
+export const runESLint = (contents, fix) => {
+  const cli = new CLIEngine({ fix })
```

Comment:
In case of `.test.js` file, we should also add `globals: [ 'describe', 'it', 'assert' ]` so that the user does not have to specify it in their eslintrc.